starlite/openapi/responses.py

```diff
@@ -182,11 +200,16 @@ def create_responses(
             route_handler=route_handler,
             generate_examples=generate_examples,
             plugins=plugins,
-        )
+        ),
     }
     exceptions = route_handler.raises or []
     if raises_validation_error and ValidationException not in exceptions:
         exceptions.append(ValidationException)
     for status_code, response in create_error_responses(exceptions=exceptions):
         responses[status_code] = response
+
+    for status_code, response in create_additional_responses(route_handler, plugins):
+        if status_code not in responses:
```

Comment:
how about raising an `ImproperlyConfiguredException` if it is already there? This happens on app bootstrap, so it should be ok and let the user know if there is a misconfiguration in place.

Reply:
Addressed in https://github.com/starlite-api/starlite/pull/478/commits/dc2b02e52214f4ef2be388ce4d5f114615bea6bb